src/components/NavBar/Navbar.jsx

```diff
@@ -1,8 +1,43 @@
 import React from 'react';
 import './Navbar.css';
+import NavLogo from '../../Images/cycle-favicon.png';
+import ProfileImg from '../../Images/pngwing.com.png';
+import {  Link } from "react-router-dom";
 
 export default function Navbar() {
   return (
-    <div>Navbar</div>
+    <>
+    <div className="sub-header">
+        <div className="login-links">
+            <Link to='/sign-in'> SignIn </Link> |
+            <Link to='/sign-up'> SignUp </Link> 
+        </div>
+    </div>
+    <nav className="navbar">
+        <div className="nav-brand">
+        <Link to='/products' className='nav-brand-name'> Faster Cycle Store </Link> 
+        <Link to=''><img src={NavLogo} alt="brand-img" className="nav-brand-img" /></Link> 
+        </div>
+       <div className="nav-contents">
+        <div className="search-bar">
+            <input type="text" placeholder="Search" className="search-input"/>
+        </div>
```

Comment:
Just a suggestion, here you can have `<input type="search" />`

Reply:
ok, I Will look on that.